test/utils.bats

```diff
@@ -119,7 +119,7 @@ teardown() {
 @test "parse_asdf_version_file should output path on project with spaces" {
   PROJECT_DIR="$PROJECT_DIR/outer space"
   mkdir -p "$PROJECT_DIR"
-  cd $outer
```

Comment:
Another test that wasn't testing what it was suppose to because of an unset variable.